pkg/nanotdf/nanotdf.go

```diff
@@ -0,0 +1,227 @@
+package nanotdf
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+type NanoTDF struct {
+	Magic              [3]byte
+	Kas                *Nanotdf_ResourceLocator
+	Binding            *Nanotdf_BindingCfg
+	SigCfg             *Nanotdf_SignatureConfig
+	Policy             *Nanotdf_Policy
+	EphemeralPublicKey *Nanotdf_EccKey
+}
+
+type Nanotdf_ResourceLocator struct {
```

Comment:
I'd suggest making these package private and renaming to remove the Nanotdf_ prefix. 
```suggestion
type resourceLocator struct {
```